android/app/src/main/java/app/organicmaps/DownloadResourcesLegacyActivity.java

```diff
@@ -44,15 +53,6 @@ public class DownloadResourcesLegacyActivity extends BaseMwmFragmentActivity
 {
   private static final String TAG = DownloadResourcesLegacyActivity.class.getSimpleName();
 
-  // Error codes, should match the same codes in JNI
-  private static final int ERR_DOWNLOAD_SUCCESS = 0;
-  private static final int ERR_DISK_ERROR = -1;
-  private static final int ERR_NOT_ENOUGH_FREE_SPACE = -2;
-  private static final int ERR_STORAGE_DISCONNECTED = -3;
-  private static final int ERR_DOWNLOAD_ERROR = -4;
-  private static final int ERR_NO_MORE_FILES = -5;
-  private static final int ERR_FILE_IN_PROGRESS = -6;
```

Comment:
What is the benefit of moving these constants into a separate file/class? Aren't they used only in this file?